codes/classical/bits/convolutional.yml

```diff
@@ -0,0 +1,44 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: convolutional
+physical: bits
+logical: bits
+
+name: 'Convolutional'
+
+introduced: '\cite{manual:{Peter Elias. Coding for noisy channels. IRE Convention Records, 3(4):37–46, 1955.}}'
+
+description: 'Convolutional codes are classical codes that are formed using generator polynomials over the finite field with two elements. The encoder slides across contiguous subsets of the input bit-string (like a convolutional neural network) evaluating the polynomials on that window to obtain a number of parity bits. These parity bits are the encoded information. There are many ways to formulate these codes'
+
+protection: 'Bit flip errors'
+
+features:
+
+  rate: 'Depends on the polynomials used. Using puncturing \cite{doi:10.12928/telkomnika.v10i4.171} removal the rate for the code can be increased from 1/n to m/n for some m, n.'
```

Comment:
Using puncturing removal \cite{...}, the rate...    ?

Reply:
Put m,n in \\( ...  \\). Also, n is not the number of bits here, correct? Maybe change the notation here and use different letters for these so as not confused quantum people.